### PR TITLE
v0.11.0-alpha release notes

### PR DESCRIPTION
**Pull request by sgrebnov**
v0.11.0-alpha release notes

### CODE REVIEW

File: docs/release_notes/v0.11.0-alpha.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+**AWS Secrets Manager as Secret Store**: The Spice runtime now supports secrets defined in AWS Secrets Manager
+
+**Manual dataset refresh via API**: The ability to run accelerated dataset refresh using API
+
+**Improved SSL support for Postgres**: ssl mode now supports `disable`, `require`, `prefer`, `verify-ca`, `verify-full` options with default mode changed to `require`.
+Added `pg_sslrootcert` parameter to allow setting custom root cert for postgres connector, `pg_insecure` parameter is no longer supported as redundant.
+
+## New Contributors
+TODO
```
Comment (sgrebnov) [inline docs/release_notes/v0.11.0-alpha.md]: Review if someone is missing

File: docs/release_notes/v0.11.0-alpha.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+**DuckDB data connector**: Use existing DuckDB database as a data source
+
+**AWS Secrets Manager as Secret Store**: The Spice runtime now supports secrets defined in AWS Secrets Manager
+
+**Manual dataset refresh via API**: The ability to run accelerated dataset refresh using API
+
+**Improved SSL support for Postgres**: ssl mode now supports `disable`, `require`, `prefer`, `verify-ca`, `verify-full` options with default mode changed to `require`.
+Added `pg_sslrootcert` parameter to allow setting custom root cert for postgres connector, `pg_insecure` parameter is no longer supported as redundant.
+
```
Comment (ewgenius) [inline docs/release_notes/v0.11.0-alpha.md:21]:
```suggestion

**Improved SSL support for MySQL**: Added `mysql_sslrootcert` parameter to allow setting custom root cert for postgres connector, `pg_insecure` parameter is no longer supported as redundant.

```

File: docs/release_notes/v0.11.0-alpha.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# Spice v0.11-alpha (Apr 15, 2024)
+
+Announcing the release of Spice.ai v0.11-alpha! 🎉
+
+The Spice.ai v0.11-alpha release brings new functionality and improves overal stability and usability.
```
Comment (lukekim) [inline docs/release_notes/v0.11.0-alpha.md]: This statement conveys no useful information. Let's add something meaningful here.

File: docs/release_notes/v0.11.0-alpha.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+## Highlights in v0.11-alpha
+
+**New data layer architecture**: The data layer was refactored to simplify creating new data components and maximize integration with and use the power of DataFusion.
```
Comment (lukekim) [inline docs/release_notes/v0.11.0-alpha.md]: What is data layer?

Comment (sgrebnov) [reply docs/release_notes/v0.11.0-alpha.md]: Updated, thank you

New architecture: The Spice runtime was refactored to simplify creating new data components and maximize integration with and use the power of DataFusion.improvements.

File: docs/release_notes/v0.11.0-alpha.md (hunk 3 above)
Comment (sgrebnov) [reply docs/release_notes/v0.11.0-alpha.md]: Updated, thank you

The Spice.ai v0.11-alpha release brings new data connector and secret store, more control over accelerated datasets refresh functionality, better security when using MySql and Postgres and overall stability improvements.

**Review by lukekim — DISMISSED** [review]

File: docs/release_notes/v0.11.0-alpha.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+**Dataset Refresh API**: Trigger a dataset refresh using the new CLI command `spice refresh` or via API.
+
+**Expanded SSL support for Postgres**: SSL mode now supports `disable`, `require`, `prefer`, `verify-ca`, `verify-full` options with the default mode changed to `require`. Added `pg_sslrootcert` parameter for setting a custom root certificate and the `pg_insecure` parameter is no longer supported.
+
+**Databricks Connect**: Choose between using Spark Connect or Delta Lake when using the Databricks data connector for improved performance.
+
+**Improved SSL support for Postgres**: ssl mode now supports `disable`, `require`, `prefer`, `verify-ca`, `verify-full` options with default mode changed to `require`.
+Added `pg_sslrootcert` parameter to allow setting custom root cert for postgres connector, `pg_insecure` parameter is no longer supported as redundant.
+
+**Internal architecture refactor**: The internal architecture of `spiceds` was refactored to simplify the creation data components and to improve alignment with DataFusion concepts.
```
Comment (phillipleblanc) [inline docs/release_notes/v0.11.0-alpha.md]:
```suggestion
**Internal architecture refactor**: The internal architecture of `spiced` was refactored to simplify the creation data components and to improve alignment with DataFusion concepts.
```

**Review by phillipleblanc — APPROVED** [review]

**Pull request merged**